scss/sections/_contact.scss

```diff
@@ -0,0 +1,69 @@
+#contact {
+	display: flex;
+	justify-content: center;
+	margin: 0 auto;
+	width: 100%;
+	max-width: 1400px;
+	.contact_container {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+		h1 {
+			font-size: 3rem;
+			text-align: center;
+			margin-bottom: 0;
+		}
+		h2 {
+			font-size: 1.5rem;
+			text-align: center;
+			margin-bottom: 4rem;
```

Comment:
[REQ]: box model dimension should be on top!
in this priority:

display
position
top/right/bottom/left etc.
height
min-height/max-height
width
min-width/max-width
padding
border
margin

....
and now all rest

this comment refers also to whole project